### DISCUSSION TITLE
Dual Receivers

### DISCUSSION THREAD

**Discussion by BHP1000** [EdgeTX/edgetx#975] [General]
How would one bind to 2 receivers and use a switch to instantly toggle between them? example operate a loader to fill a dump truck then instantly switch to the truck to reposition and/or drive off. Also if possible I would like to use 2 to 3 RC rigs at the same time kind of like a convoy profile.

Thank you for your time,
 BHP1000

**Comment by pfeerick** [comment]
You could use multiple receivers like the X8R where you can bind two, giving you 16 channels (1-8 on one [with telemetry], 9-16 on the other [without telemetry]) Then on your transmitter change your input => channel assignments based on switch, effectively switching between the two receivers? I don't know any that let you have access to more than two though. There are other systems like Spectrum DSM2 which can be tricked to 'multi-bind' by binding all your receivers at the same time, but then you would need to figure out some way to determine which receivers output should be 'enabled' ie. a spare channel that goes high med low (for three devices) and cuts power to drivetrains when not active.

**Reply by BHP1000** [reply]
That is one way that I did not consider thanks. However, the rc's tend to use more then 8 ch's. And I don't have multiple x8r recivers.
